### PR TITLE
add debugging env for GC time window

### PR DESCRIPTION
**Pull request by wy65701436**
For debugging, the tester/users wants to run GC to delete the removed artifact immediately instead of waitting for two hours, add the env(_GC_BLOB_TIME_WINDOW) to meet this.

Signed-off-by: wang yan <wangyan@vmware.com>

### CODE REVIEW

File: src/common/const.go [hunk 1]
```diff
@@ -154,6 +154,6 @@ const (
 	CountPerProject       = "count_per_project"
 	StoragePerProject     = "storage_per_project"
 
-	// ForeignLayer
-	ForeignLayer = "application/vnd.docker.image.rootfs.foreign.diff.tar.gzip"
+	// DefaultGCBlobTimeWindow is the reserve blob time window used by GC, default is 2 hours
+	DefaultGCBlobTimeWindow = 2
```
Comment (heww) [inline src/common/const.go:158]: DefaultGCBlobTimeWindowHours = int64(2)

File: src/core/config/config.go [hunk 2]
```diff
@@ -478,3 +479,15 @@ func GetPermittedRegistryTypesForProxyCache() []string {
 	}
 	return strings.Split(types, ",")
 }
+
+// GetGCBlobTimeWindow returns the reserve time window of blob.
+func GetGCBlobTimeWindow() int64 {
+	// the env is for testing/debugging. For production, Do NOT set it.
+	if env, exist := os.LookupEnv("_GC_BLOB_TIME_WINDOW"); exist {
```
Comment (heww) [inline src/core/config/config.go:486]: `_GC_BLOB_TIME_WINDOW_HOURS`. Do we need `BLOB` in this env ? or `_GC_TIME_WINDOW_HOURS`?

Comment (reasonerjt) [reply src/core/config/config.go:486]: +1
and I don't think we need the first underscore

Comment (wy65701436) [reply src/core/config/config.go]: updated

File: src/common/const.go (hunk 1 above)
Comment (wy65701436) [reply src/common/const.go]: updated.

**Pull request merged**